src/webui/www/private/scripts/client.js

```diff
@@ -230,62 +230,40 @@ window.addEventListener("DOMContentLoaded", () => {
     buildLogTab();
     MochaUI.initializeTabs("mainWindowTabsList");
 
+    setStatusFilter = function(name) {
+        selectedStatus = name;
+        highlightSelectedStatus();
+        LocalPreferences.set("selected_filter", name);
```

Comment:
Strange to see this operation placed here.
It should either be above `selectedCategory = Number(hash);` or below `updateMainData();`.

Please adjust other similar instance too. (setCategoryFilter, setTagFilter, setTrackerFilter)